src/altinn-app-frontend/src/utils/validation.test.ts

```diff
@@ -1,30 +1,34 @@
-import { createElement } from 'react';
-import { getInitialStateMock } from '../../__mocks__/initialStateMock';
-import { getMockValidationState } from '../../__mocks__/validationStateMock';
+import {createElement} from 'react';
+import {getInitialStateMock} from '../../__mocks__/initialStateMock';
+import {getMockValidationState} from '../../__mocks__/validationStateMock';
 import * as oneOfOnRootSchema from '../../__mocks__/json-schema/one-of-on-root.json';
 import * as refOnRootSchema from '../../__mocks__/json-schema/ref-on-root.json';
 import * as complexSchema from '../../__mocks__/json-schema/complex.json';
 
 import type {
-  IValidationIssue,
-  IValidations,
-  IRepeatingGroups,
-  IRuntimeState,
   IComponentBindingValidation,
   IComponentValidations,
   ILayoutValidations,
+  IRepeatingGroups,
+  IRuntimeState,
+  IValidationIssue,
+  IValidations,
 } from 'src/types';
-import type { ILayoutComponent, ILayoutGroup } from 'src/features/form/layout';
-
-import { Severity } from 'src/types';
-import { getParsedLanguageFromKey } from '../../../shared/src';
-import { createRepeatingGroupComponents } from 'src/utils/formLayout';
-import { getParsedTextResourceByKey } from 'src/utils/textResource';
+import {Severity} from 'src/types';
+import type {ILayoutComponent, ILayoutGroup} from 'src/features/form/layout';
+import {getParsedLanguageFromKey} from '../../../shared/src';
+import {createRepeatingGroupComponents} from 'src/utils/formLayout';
+import {getParsedTextResourceByKey} from 'src/utils/textResource';
 import * as validation from './validation';
-import { mapToComponentValidations } from './validation';
+import {mapToComponentValidations} from './validation';
```

Comment:
I guess this is because we are not all using the same editors, and the same formatters.

We need to get this fixed, and set up formatting via prettier on commit :) I would say just leave it for now, and lets format all the code when this is in place.

Reply:
Not really, no... I assumed it was `eslint --fix`, but it turns out it was Intellij automatically formatting and sorting the imports for me. I've adjusted my local code style now to avoid this in the future.

Reply:
@haakemon Prayers almost answered by #77! :blush: